### DISCUSSION TITLE
where is overlapping detection in v1?

### DISCUSSION THREAD

**Discussion by bojone** [mapequation/infomap#61] [Q&A] [answered]
as the title

**Comment by danieledler** [comment, marked as answer]
Sorry for the late reply @bojone, potentially overlapping modules need higher-order network input, generally represented as a [state network](https://www.mapequation.org/infomap/#InputStates) in Infomap.

In the old 0.x version, we had the flag `--overlapping` which artificially generated a second-order network from a first-order ordinary network by assuming that two bigrams (ordinary links) `a b` and `b c` forms a trigram `a b c`, making the flow between two nodes dependent on where it came from from step before. However, that information doesn't exist in the network itself so it's not a recommended way. If you want overlapping modules, you should input a higher-order network. What kind of data do you have?
